docs/gitbook/quick-start/docker.md

```diff
@@ -42,6 +42,9 @@ redis:
     global: consoleme-redis
   db: 0
   port: 6379
+  
```

Comment:
Hi @eamonnfaherty! Thank you for the PR! The config variables you added here are actually part of the previous step. In that step, when you modify one of the example configs, or create a similar config to the example configs, it will include all relevant tornado configurations needed. For example, [this](https://github.com/Netflix/consoleme/blob/master/example_config/example_config_base.yaml) base config includes  relevant tornado settings [here](https://github.com/Netflix/consoleme/blob/master/example_config/example_config_base.yaml#L8-L14).
With that said, if it was confusing, maybe instead we can add some more clarifications to the previous step instead?

Also, for convenience, you can go to `<consoleme_url>/generate_config` to generate a config once you have ConsoleMe running in dev mode. For example, on the demo site, you can go to https://demo.consolemeoss.com/generate_config.